plugins/gcloud/test/samson_gcloud/samson_plugin_test.rb

```diff
@@ -111,6 +111,43 @@ def expects_sleep(times)
         end
       end
     end
+
+    describe "gcloud vulnerabilty scanning" do
+      with_env GCLOUD_IMAGE_SCANNER: "true", GCLOUD_ACCOUNT: 'acc', GCLOUD_PROJECT: 'example'
+
+      def fire
+        Samson::Hooks.fire(:ensure_docker_image_has_no_vulnerabilities, stage, image)
+      end
+
+      let(:image) { +'foo.com/example/bar' }
```

Comment:
what is that "+" here ? Is that intended ?

Reply:
to make variable nonfrozen. because `# frozen_string_literal: true` is at top of file.